packages/core/src/amazonqFeatureDev/util/files.ts

```diff
@@ -19,6 +19,8 @@ import { TelemetryHelper } from './telemetryHelper'
 
 const getSha256 = (file: Buffer) => createHash('sha256').update(file).digest('base64')
 
+export const amazonqFeatureDevMaxRepoSize = 200 * 1024 * 1024 // 200 MB
```

Comment:
can the existing constant be used? https://github.com/aws/aws-toolkit-vscode/blob/e52e000a9116a1a718bb1b54c1c4172c9cd38efe/packages/core/src/amazonqFeatureDev/constants.ts#L18

I see a lot of global constants being created, but rarely used more than once.

Reply:
Yes! good catch.